ups-connector-demo/pom.xml

```diff
@@ -11,7 +11,7 @@
   <pluginRepositories>
     <pluginRepository>
       <id>sonatype</id>
-      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+      <url>https://central.sonatype.com/repository/maven-snapshots</url>
```

Comment:
@quanpham-axonivy I think you triggered these PRs. Was it an accident that you merged PRs like these using the "Octopus-AxonIvy" account? 
Merging things which such shared accounts is usually bad. We obfuscate the history and asking back in case of issues with the change to the guy who introduced it is impossible. 
So I'd vote to personalize such changes in the future, at least the 'assignee' should be set to the user that actually has run the change. 
Can we adapt the script accordingly? Or did we already meanwhile?